pay/contracts/v1_upfront_pay/HectorPayFactory.sol

```diff
@@ -147,44 +111,9 @@ contract HectorPayFactory is IHectorPayFactory, Ownable {
 
     /* ======== VIEW FUNCTIONS ======== */
 
-    /**
-      @notice Query the address of the Hector Pay contract for `_token` and whether it is deployed
-      @param _token An ERC20 token address
-      @return isDeployed Boolean denoting whether the contract is currently deployed
-      */
     function isDeployedHectorPayContractByToken(
         address _token
     ) external view returns (bool isDeployed) {
         isDeployed = getHectorPayContractByToken[_token] != address(0);
     }
-
-    /* ======== SUBSCRIPTION POLICY FUNCTIONS ======== */
-
-    function pauseStreams(Stream[] calldata streams) external {
-        uint256 length = streams.length;
-        for (uint256 i = 0; i < length; i++) {
-            Stream memory stream = streams[i];
-            IHectorPay(stream.payContract).pauseStreamBySubscription(
-                stream.from,
-                stream.to,
-                stream.amountPerSec,
-                stream.starts,
-                stream.ends
-            );
-        }
-    }
-
-    function resumeStreams(Stream[] calldata streams) external {
```

Comment:
@bejaxer curious why func resumeStreams is removed? This could potentially affect the dApp...